loader/src/reference_binder.py

```diff
@@ -223,3 +168,44 @@ def load_cached_references(self, target_ref: str) -> DefaultDict[tuple, list]:
             (source_ref, ref) = json.loads(element)
             pending_refs[tuple(source_ref)].append(ref)
         return pending_refs
+
+    @staticmethod
+    def identifier_to_key(resource_type, identifier):
+        value = identifier.get("value")
+        system = identifier.get("system")
+        return f"{resource_type}:{value}:{system}"
+
+    @staticmethod
+    def partial_identifier(identifier):
+        return {
+            "identifier.value": identifier.get("value"),
+            "identifier.system": identifier.get("system")
+        }
+
+    def build_find_predicate(self, resource_ids, reference_path, identifier, is_array):
```

Comment:
What about making this one static and using ReferenceBinder.partial_identifier?
Just for the sake of symmetry 🤷‍♂️

Reply:
Method removed